server/core/include/irods/configuration_hook_manager.hpp

```diff
@@ -0,0 +1,17 @@
+#ifndef CONFIGURATION_HOOK_MANAGER_HPP
+#define CONFIGURATION_HOOK_MANAGER_HPP
+
+#include <vector>
+#include <string>
+#include <nlohmann/json.hpp>
```

Comment:
Please reorder to:
```cpp
#include <nlohmann/json.hpp>

#include <vector>
#include <string>
```

We include things in the following order:
1. iRODS headers
2. Third-party headers (grouped for readability)
3. System headers
4. C headers (`<cstring>`)
5. C++ STL headers